src/components/qoutes.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+
+function Qoutes() {
+  const [quotes, setQuotes] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  const fetchQuotes = async () => {
+    const category = 'happiness';
+    try {
+      const response = await axios({
+        url: `https://api.api-ninjas.com/v1/quotes?category=${category}`,
+        method: 'GET',
+        headers: { 'X-Api-Key': 'JIMZKaMLieUwg8xIF54R1w==C0w5bqNgmdl8Ao0c' },
+        contentType: 'application/json',
+      });
+      const data = response.data[0];
+      setQuotes(data);
+      setLoading(false);
+    } catch (err) {
+      setError(err.message);
+    }
+  };
+  useEffect(() => {
+    fetchQuotes();
+  }, [quotes]);
```

Comment:
- [ ] The requirement is that the quote is fetched and rendered once, but yours continuously makes a fetch and keeps re-rendering different quotes, like a loop. Kindly update your useEffect to only fetch the quotes, once the component is mounted.

  ```
  useEffect(() => {
    fetchQuotes();
  }, []);
  ```